<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GraphicsWindow</class>
 <widget class="QWidget" name="GraphicsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1120</width>
    <height>630</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1120</width>
    <height>630</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel{
	font-size: 20px;
	color: rgb(65, 74, 76);
}</string>
  </property>
  <widget class="PlotWidget" name="graph_big" native="true">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>40</y>
     <width>981</width>
     <height>471</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">PlotWidjet{
	Background-color: rgb(255, 255, 255)
}</string>
   </property>
  </widget>
  <widget class="QPushButton" name="go_back_btn_graph">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>570</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color: rgb(52, 59, 41);
	color: rgb(255, 255, 255);
	border-radius: 5px;
	font-size: 17px;
}
QPushButton:pressed {
	background-color: rgb(39, 45, 30);
}</string>
   </property>
   <property name="text">
    <string>вернуться</string>
   </property>
  </widget>
  <widget class="QPushButton" name="go_back_btn_graph_2">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>530</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color: rgb(52, 59, 41);
	color: rgb(255, 255, 255);
	border-radius: 5px;
	font-size: 17px;
}
QPushButton:pressed {
	background-color: rgb(39, 45, 30);
}</string>
   </property>
   <property name="text">
    <string>обновить</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QWidget</extends>
   <header location="global">pyqtgraph</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
